homeassistant/components/calendar/__init__.py

```diff
@@ -261,8 +262,10 @@ def _validate_rrule(value: Any) -> str:
     extra=vol.ALLOW_EXTRA,
 )
 
-SERVICE_LIST_EVENTS: Final = "list_events"
-SERVICE_LIST_EVENTS_SCHEMA: Final = vol.All(
+LEGACY_SERVICE_LIST_EVENTS: Final = "list_events"
```

Comment:
I guess this is a matter of preference but I don't understand why these are renamed to `LEGACY_...`

Reply:
I want it to be easier to understand just by looking at the code/tests, that this is a service which is going to be removed. With an IDE you can mouseover and get the docstring but with new new variable a few seconds reading the code are saved. I also think it makes it easier to find in half a year when we are going to remove it.

Reply:
Don't see it but it's not going to block this PR from being merged in.